### PR TITLE
Ubuntu Docker Script for Building Client-SDK

### PR DESCRIPTION
**Pull request by DukeDavis12**
1. The Docker script is able to build the local copy of source code,
     by mounting the local copy into the container and building the mounted local copy.

  2. The script is able to build the latest upstream copy,
     by cloning and building the upstream in /tmp folder of the container and
     finally copying the executables into the mounted local copy.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

**Comment by trbehera** [comment]
@AdithyaBaglody @nbmaiti This PR is created to create a dockerized build environment for building the source code. Please review.

File: docker_build/build.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```
Comment (AdithyaBaglody) [inline docker_build/build.sh:1]: why not move all this inside the Dockerfile itself. when we compose we will have all the dependency ready. So, only the build scripts can be part of the execution. its too slow to do this each time.

File: docker_build/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Intel Corporation
```
Comment (AdithyaBaglody) [inline docker_build/Dockerfile:1]: This folder can be moved inside utils.

Comment (DukeDavis12) [reply docker_build/Dockerfile]: Moved the docker_build folder to utils/

File: docker_build/build.sh (hunk 1 above)
Comment (DukeDavis12) [reply docker_build/build.sh]: Moved all dependencies to Dockerfile. Now when we re-run the container, it simply executes the build scripts.

**Pull request merged**